qpc_tools/utils.py

```diff
@@ -170,15 +170,21 @@ def get_password(args_dictionary):
     :param args_dictionary: the dictionary containing the args and values
     :returns: the dictionary with updated passwords
     """
-    arg_prompt = OrderedDict([
-        ('rh_registry_username', 'Enter registry.redhat.io username: '),
-        ('rh_registry_password', 'Enter registry.redhat.io password: '),
-        ('server_password', 'Enter server password: '),
-        ('db_password', 'Enter database password: ')
-    ])
+    if args_dictionary.get('registry_no_auth'):
+        arg_prompt = OrderedDict([
+            ('server_password', 'Enter server password: '),
+            ('db_password', 'Enter database password: ')
+        ])
+    else:
+        arg_prompt = OrderedDict([
+            ('registry_username', 'Enter registry.redhat.io username: '),
```

Comment:
what if they pass in a different url for the registry? This prompt won't make sense.

Reply:
Oh that is a really good catch!! I will change it to registry.